### ISSUE TITLE
[bug] myks mandates a vendir.yaml structure whereby the entire path is held at directory level

### ISSUE THREAD

**Issue by fritzduchardt** [mykso/myks#171]
Composite paths such as this lead to an empty vendor directory:

```
apiVersion: vendir.k14s.io/v1alpha1
kind: Config
directories:
  - path: "charts/"
    contents:
      - path: #@ app.chart.name
        helmChart:
          name: #@ app.chart.name
          version: #@ app.chart.version
          repository:
            url: #@ app.chart.url
```

**Comment by Zebradil** [comment]
It was first reported in #155 and was fixed in #156 :-)

**Comment by fritzduchardt** [comment]
Overtaken by evens. Should learn to read issues before taking action.

**Comment by fritzduchardt** [comment]
> It was first reported in #155 and was fixed in #156 :-)

@Zebradil just as a side-node on this: looking at the logic, I would suspect this to not clean up on composite paths once we reintroduce the directory flag. The logic would have to concatenate paths at directory level as well as contents level.

**Comment by Zebradil** [comment]
We can change the logic when needed.
